### PR TITLE
Fix/Issues related to styling

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: components/graph-line.vue [hunk 1]
```diff
@@ -21,6 +21,7 @@
       <v-chart
         :options="graphData()"
         :autoresize="true"
+        height="100%"
```
Comment (SiggyF) [inline components/graph-line.vue]: in css?

File: components/graph-line.vue [hunk 2]
```diff
@@ -217,16 +219,20 @@ export default {
...
+.echarts {
+  width: 100%;
+  height: 100%;
+}
+
 .graph-line {
   position: relative;
-
-  --caption-height: 4rem;
+  --caption-height: 3rem;
```
Comment (SiggyF) [inline components/graph-line.vue]: do we use postcss?

File: components/graph-line.vue (hunk 1 above)
Comment (CindyvdVries) [reply components/graph-line.vue]: it's an attribute for the vue echarts comp

File: components/graph-line.vue (hunk 2 above)
Comment (CindyvdVries) [reply components/graph-line.vue]: yes

**Pull request merged**